src/algebra/module/submodule/pointwise.lean

```diff
@@ -179,6 +179,13 @@ open_locale pointwise
 lemma smul_mem_pointwise_smul (m : M) (a : α) (S : submodule R M) : m ∈ S → a • m ∈ a • S :=
 (set.smul_mem_smul_set : _ → _ ∈ a • (S : set M))
 
+/-- See also `submodule.smul_bot`. -/
+@[simp] lemma smul_bot' (a : α) : a • (⊥ : submodule R M) = ⊥ := map_bot _
+/-- See also `submodule.smul_sup`. -/
+lemma smul_sup' (a : α) (S T : submodule R M) : a • (S ⊔ T) = a • S ⊔ a • T := map_sup _ _ _
+lemma smul_span (a : α) (s : set M) : a • span R s = span R (a • s) := map_span _ _
+lemma span_smul (a : α) (s : set M) : span R (a • s) = a • span R s := eq.symm (span_image _).symm
```

Comment:
Can you either drop this lemma or write `submonoid.closure_smul`/`subgroup.closure_smul`/... ?

Reply:
I added this lemma because for some reason we decided to have lemmas for both directions of the `map` spelling for `submodule` but not the other subobjects.

Reply:
Hmm... weird, but okay.

maintainer merge

Reply:
Note the weird proof is deliberate; it means that if we decide to remove `span_image`, CI will suggest to remove this lemma too.